app/views/virtual_workshops/show.html.haml

```diff
@@ -2,33 +2,36 @@
 
 = render partial: 'virtual_workshops/meta_tags', locals: { workshop: @workshop }
 
-.container-fluid.stripe.reverse
-  .row
-    .col
-      %h2
-        = t('workshops.virtual.title', chapter: @workshop.chapter.name)
-        %br
-        %small #{humanize_date(@workshop.date_and_time, @workshop.ends_at, with_time: true)} #{@workshop.distance_of_time}
-      - if @workshop.date_and_time.past?
-        %span.badge.bg-danger= t('messages.already_taken_place')
+.stripe.reverse
```

Comment:
```suggestion
.py-4.py-lg-5.bg-light
```


Let's update all the `.stripe.reverse` classes to `.py-4.py-lg-5.bg-light` in this file as well!